operate/pom.xml

```diff
@@ -257,6 +256,19 @@
         </plugins>
       </build>
     </profile>
-  </profiles>
 
+    <!-- 
+      Integration tests are run per default, but having it in a profile allows us to exclude it easily.
+      That is useful for example for creating releases.
+       -->
```

Comment:
🔧 Following up on the discussion I would suggest the following to make clear we want to ignore the whole module not just skip test runs.
```suggestion
    <!-- 
      The QA module is enabled per default, but having it in a profile allows us to exclude it easily.
      That is useful for example for creating releases.
       -->
```